java/arcs/android/storage/service/BindingContext.kt

```diff
@@ -190,7 +217,47 @@ class BindingContext(
         }
     }
 
+    /**
+     * Increment an active message count and send it to the active messages count flow.
+     *
+     * Calls to [idle] will wait for this value to reach 0 before checking store activity.
+     *
+     * Avoid using this for calls to [idle] itself, so as to not increment the activity counter.
+     */
+    private fun launchNonIdleAction(action: suspend () -> Unit) {
+        activeMessagesCountFlow.increment()
+        // [CoroutineStart.UNDISPATCHED] here to ensures that the actions are started in order.
```

Comment:
Should this comment be on line 257 instead?

Reply:
argh, yes.